stellargraph/layer/watch_your_step.py

```diff
@@ -202,7 +202,7 @@ def build(self):
             name="WATCH_YOUR_STEP_RIGHT_EMBEDDINGS",
         )(vectors_left)
 
-        sigmoids = tf.keras.activations.sigmoid(outer_product)
+        # sigmoids = tf.keras.activations.sigmoid(outer_product)
```

Comment:
Maybe instead of leaving commented out code, we could delete it, or at least explain what's going on.

Reply:
missed this, will delete